cmd/kubectl-gadget/seccompadvisor.go

```diff
@@ -67,7 +67,7 @@ func init() {
 		"The trace output mode, possibles values are terminal and seccomp-profile.")
 	seccompAdvisorStartCmd.PersistentFlags().StringVar(&seccompProfileName,
 		"seccomp-profile-name", "",
-		"Name of the seccomp profile to be created when using --output-mode=seccomp-profile.\nNamespace can be specified by using namespace/profile-name.")
+		"Prefix name of the seccomp profile to be created when using --output-mode=seccomp-profile.\nNamespace can be specified by using namespace/profile-name.")
```

Comment:
Should `seccomp-profile-name` also be renamed to `seccomp-profile-prefix` or `seccomp-profile-prefix-name`?

Reply:
Good point, I was also thinking about it but I found `seccomp-profile-prefix` or `seccomp-profile-prefix-name` too long. What about `profile-prefix-name`? In the end, the seccomp is implicit, don't you think?

cc @eiffel-fl

Reply:
I agree with `profile-prefix-name`, it is a good compromise between name length and information giving.

Reply:
I agree as well. Seccomp can be dropped.

Reply:
Done